data/json/recipes/armor/suit.json

```diff
@@ -852,11 +852,12 @@
     "category": "CC_ARMOR",
     "subcategory": "CSC_ARMOR_SUIT",
     "skill_used": "tailor",
+    "proficiencies": [ { "proficiency": "prof_closures" } ],
     "difficulty": 4,
     "time": "6 h",
     "autolearn": true,
     "book_learn": [ [ "textbook_armwest", 3 ] ],
-    "using": [ [ "sewing_standard", 30 ] ],
+    "using": [ [ "sewing_standard", 35 ], [ "fastener_large", 1 ] ],
```

Comment:
Wouldn’t this need multiple fasteners or a zipper?

Reply:
I think you meant the XL version... And it does, thanks for the remainder! I knew I was forgetting something.